### PR TITLE
(DPS-3134) Extract webhook payloads from Configuration API

### PR DESCRIPTION
**Pull request by ale-kacperczyk**
# 🚀 Links

Replace the placeholder data in links 👇

- [Jira](https://livechatinc.atlassian.net/browse/DPS-3134)

# 📓 Description

I've created a new versioned document, **Webhooks**, only for webhook payloads. (v3.1-v3.4)

### CODE REVIEW

**Review by bkubiak — APPROVED** [review]
I like the changes :) The only problems are:
* links from agent-api (web) to webhooks
* unnecessary left sidebar links (specifics, payload) related to every webhook

File: src/pages/messaging/agent-chat-api/v3.2/index.mdx [hunk 1]
```diff
@@ -2305,12 +2305,12 @@ Creates a new [Customer](#customer) user type.
...
-| **Method URL**         | `https://api.livechatinc.com/v3.2/agent/action/create_customer`       |
-| **Required scopes**    | `customers:rw`                                                        |
-| **RTM API equivalent** | [`create_customer`](rtm-reference/#create-customer)                   |
-| **Webhook**            | [`customer_created`](/management/configuration-api/#customer_created) |
+|                        |                                                                   |
+| ---------------------- | ----------------------------------------------------------------- |
+| **Method URL**         | `https://api.livechatinc.com/v3.2/agent/action/create_customer`   |
+| **Required scopes**    | `customers:rw`                                                    |
+| **RTM API equivalent** | [`create_customer`](rtm-reference/#create-customer)               |
+| **Webhook**            | [`customer_created`](/management/webhooks/v3.2/#customer_created) |
```
Comment (ale-kacperczyk) [inline src/pages/messaging/agent-chat-api/v3.2/index.mdx:2313]: This webhook doesn't seem to exist in Webhooks v3.2. Debug why.

**Comment by ale-kacperczyk** [comment]
One more thing to consider before we merge: what about changelogs? Should we create a separate changelog for this doc or log the changes in the existing Changelog with links to the new doc? @bkubiak

File: src/pages/management/webhooks/v3.2/index.mdx [hunk 2]
```diff
@@ -0,0 +1,1319 @@
...
+### `chat_deactivated`
+
+Informs that a chat was deactivated by closing the currently open thread.
+
+#### Specifics
+
+|                        |                                                                                                                                                                             |
+| ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Action**             | `chat_deactivated`                                                                                                                                                          |
+| **Push equivalent in** | [`Agent Chat API`](/messaging/agent-chat-api/v3.3/rtm-reference/#chat_deactivated) [`Customer Chat API`](/messaging/customer-chat-api/v3.3/rtm-reference/#chat_deactivated) |
```
Comment (ale-kacperczyk) [inline src/pages/management/webhooks/v3.2/index.mdx:110]: This docs has v3.3 webhooks. Copy-paste error.

**Pull request merged**